tests/test_display.py

```diff
@@ -413,6 +423,44 @@ def test_animshow(self, as_rgb, channel_idx, batch_idx, is_complex, mini_vid):
                 po.animshow(vid, as_rgb=as_rgb, channel_idx=channel_idx,
                             batch_idx=batch_idx, plot_complex=is_complex)
 
+    def test_update_plot_shape_fail(self):
+        im = po.load_images(op.join(DATA_DIR, 'nuts.pgm'))
+        fig = po.imshow(im)
+        # update_plot expects 3 or 4d data -- this checks that it fails
+        # (because im.squeeze() is 2d) and raises the proper exception
+        with pytest.raises(Exception):
+            try:
+                po.update_plot(fig.axes[0], im.squeeze())
+            except Exception as e:
+                assert '3 or 4 dimensional' in e.args[0], "WRONG EXCEPTION"
+                raise e
+
+    def test_synthesis_plot_shape_fail(self):
+        im = po.load_images(op.join(DATA_DIR, 'nuts.pgm'))
+
+        class DumbModel(po.simul.PooledRGC):
+            def forward(self, *args, **kwargs):
+                output = super().forward(*args, **kwargs)
+                return output.reshape(output.numel())
+        model = DumbModel(.5, im.shape[2:]).to(DEVICE)
+        met = po.synth.Metamer(im, model)
+        met.synthesize(max_iter=3, store_progress=True)
+        # Synthesis plot_synthesis_status and animate expect 3 or 4d data --
+        # this checks that it fails (because im.squeeze() is 2d) and raises the
```

Comment:
check that "it" fails?  Can you make it slightly more clear what these tests are testing for?

Reply:
Sure thing.